### PR TITLE
Add mGBA

### PR DESCRIPTION
**Pull request by BlackPowerade**
Only stable for right now. the development download link does not have the latest build, and I have no idea how to reliably get the latest build off of travis CI.

### CODE REVIEW

**Comment by BlackPowerade** [comment]
Contacted owner, got get links fixed. mgba-dev package is ready, but I don't know how fetch both the major version and build number when they are on different lines.
`"regex": "latest release: (?<major>[\\d.]+)/s(?<build>[\\d]+)-"`
Something like this?

File: bucket/mgba-dev.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        "savegame",
+        "savestate",
+        "screenshot",
+        "shaders",
+        "qt.ini",
+        "config.ini"
+    ],
+    "checkver": {
+        "url": "https://mgba.io",
+        "regex": "(?<build>[\\d]+)-",
```
Comment (r15ch13) [inline bucket/mgba-dev.json]:
```suggestion
        "regex": "Release:\\s+<[^>]+>(?<version>[\\d.]+)</a>[\\s\\S]+Version:\\s+<[^>]+>(?<build>[a-f\\d-]+)</a>",
```

File: bucket/mgba-dev.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        "replace": "0.7.3-${1}"
```
Comment (r15ch13) [inline bucket/mgba-dev.json]:
```suggestion
        "replace": "${1}-${2}"
```

**Comment by BlackPowerade** [comment]
Thanks @r15ch13

**Comment by Calinou** [comment]
Thanks!

**Pull request merged**